### PR TITLE
fix(crafter): std_dev bug

### PR DESCRIPTION
**Pull request by kujta1**
fix(crafter): std_dev can only have values larger than zero (0)

### CODE REVIEW

**Review by JoanFM — CHANGES_REQUESTED** [review]
Hey @kujta1 ,

thank you for your contribution, could you please add a test for this case and bump the version number so that the tests are actually executed?

**Comment by kujta1** [comment]
The tests and the version numbers have been updated

**Comment by JoanFM** [comment]
Hey @kujta1 ,

thank you very much for the contribution, unfortunately your commit messages do not seem to match our guidelines.

Please refer to:

- [Commit Message Guideline for the First Time Contributor](https://github.com/jina-ai/jina/issues/553)
- [Contributing Guideline](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md)

**Review by JoanFM — CHANGES_REQUESTED** [review]
Hey @kujta1, could you please keep only one change in the `PR` (the one with `ImageNormalizer`). I see other hub executors being updated

**Comment by nan-wang** [comment]
@kujta1 hi, there. Please bump the `version` to `0.0.14`. https://github.com/jina-ai/jina-hub/blob/master/crafters/image/ImageNormalizer/manifest.yml#L11

**Comment by nan-wang** [comment]
recheckcla

**Comment by nan-wang** [comment]
**Jina CLA check**<br/>
  ❤️ Thank you for your pull request. It looks like this is your first contribution to an open source project maintained by Jina AI Limited. Before we can look at your pull request, we kindly ask that you sign our [Contributor License Agreement](https://github.com/jina-ai/jina/blob/master/.github/CLA.md). You can sign it by commenting in format below.
  - - -
  ***I have read the CLA Document and I hereby sign the CLA***
  - - -

**Comment by kujta1** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by nan-wang** [comment]
@kujta1 The commit messages do not meet your contribution guidelines, please refer to:
- [Commit Message Guideline for the First Time Contributor](https://github.com/jina-ai/jina/issues/553)
- [Contributing Guideline](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md)


https://github.com/jina-ai/jina-hub/pull/3842/checks?check_run_id=1941803761
```
⧗   input: change version number
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

✖   found 2 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint

⧗   input: tests for standard deviation of ImageNormalizer greater than zero
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

✖   found 2 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint

⧗   input: fix std_dev bug. Std_dev can only have values larger than zero (0). If it has value zero it may result in division by zero when an image is normalzied
✖   header must not be longer than 72 characters, current length is 150 [header-max-length]
✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

✖   found 3 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
```

File: rankers/SimpleAggregateRanker/__init__.py [hunk 1]
```diff
@@ -8,7 +8,6 @@ class SimpleAggregateRanker(Chunk2DocRanker):
     of the matched doc from the matched chunks.
     For each matched doc, the score is aggregated
     from all the matched chunks belonging to that doc.
-
```
Comment (florian-hoenicke) [inline rankers/SimpleAggregateRanker/__init__.py:11]: please undo this change

File: crafters/image/ImageReader/manifest.yml [hunk 2]
```diff
@@ -7,6 +7,6 @@ author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub/blob/master/crafters/image/ImageReader/README.md
-version: 0.0.14
```
Comment (florian-hoenicke) [inline crafters/image/ImageReader/manifest.yml:10]: image reader version does not have to be updated. please update the image normalizer version instead

Comment (nan-wang) [reply crafters/image/ImageReader/manifest.yml:10]: good catch!

**Comment by nan-wang** [comment]
This issue is stale and closed because it has been open for more than 20 days with no activity.
